v2/test/sample_test.py

```diff
@@ -96,7 +96,8 @@ def main(
             'samples_destination': f'{gcr_root}/v2-sample-test',
             'kfp_host': host,
         },
-        experiment_name=experiment
+        experiment_name=experiment,
+        enable_caching=False,
```

Comment:
I'm not sure whether `enable_caching` is needed here. I think this pipeline still uses v1 kfp.

Reply:
Reverted.